xla/service/gpu/triton_call.cc

```diff
@@ -44,7 +44,12 @@ TritonCall TritonCall::Parse(absl::string_view backend_config,
       attrs.getAs<mlir::IntegerAttr>("num_stages").getValue().getSExtValue();
   auto num_warps =
       attrs.getAs<mlir::IntegerAttr>("num_warps").getValue().getSExtValue();
-  return TritonCall{std::move(name), std::move(ir), num_stages, num_warps,
+  int64_t waves_per_eu = 0;
+  auto attr_wpeu = attrs.getAs<mlir::IntegerAttr>("waves_per_eu");
```

Comment:
I thought we were gonna move this to serialized_data? (can't remember the exact name of that field)

Reply:
yes, this PR is no longer needed https://github.com/ROCm/xla/pull/181